code/modules/mob/living/carbon/human/species/emote-monkey.dm

```diff
@@ -2,6 +2,10 @@
 	species_type_allowed_typecache = list(/datum/species/monkey)
 	keybind = FALSE
 
+/datum/emote/living/carbon/human/primate/New()  //Monkey's are blacklisted from human emotes on emote.dm, we need to not block the new emotes below
+	. = ..()
+	species_type_blacklist_typecache -= /datum/species/monkey
```

Comment:
Will null this out, originally I figured it would be future proofing but given that this runs once and isn't stored (I was unaware) it's probably just best to have it set to NULL.